PyDSS/pyControllers/Controllers/PvDynamic.py

```diff
@@ -0,0 +1,299 @@
+from  PyDSS.pyControllers.pyControllerAbstract import ControllerAbstract
+
+try:
+    from pvder.simulation_utilities import SimulationResults
+    from pvder.dynamic_simulation import DynamicSimulation
+    from pvder.simulation_events import SimulationEvents
+    from pvder.grid_components import Grid
+    from pvder.DER_wrapper import DERModel
+except:
```

Comment:
I suggest changing this to `except ImportError`. As it stands, if there is another type of exception raised in one of these modules, the user will get a misleading error message.

Reply:
FIXED